src/components/application_manager/src/resumption/resumption_sql_queries.cc

```diff
@@ -428,7 +428,7 @@ const std::string kUpdateHMILevel =
 
 const std::string kUpdateIgnOffCount =
     "UPDATE `application` "
-    "SET `ign_off_count` = `ign_off_count` - 1"
+    "SET `ign_off_count` = `ign_off_count` - 1 "
```

Comment:
@dev-gh no. I see ` "SET `ign_off_count` = `ign_off_count` + 1"` (without space at the end) some below